frontend/apps/marketing/src/components/common/types/index.ts

```diff
@@ -1,3 +1,5 @@
+export type ComponentSize = 'xs' | 's' | 'm' | 'l';
```

Comment:
This can be used with the Paragraph and Link components when those are refactored.